config.toml

```diff
@@ -23,8 +23,8 @@ juice_logo_path = "logo_pfb.svg"
 juice_extra_menu = [
     # { title = "Overview", link = "/" },
     # { title = "Community", link = "/community" },
-    { title = "Docs ⍈", link = "https://peopleforbikes.github.io/docs" },
-    { title = "Github ⍈", link = "https://github.com/PeopleForBikes" }
+    { title = "Docs ↗", link = "https://peopleforbikes.github.io/docs" },
+    { title = "Github ↗", link = "https://github.com/PeopleForBikes" }
```

Comment:
Great UTF-8 symbols for external links! 👍

Reply:
It'll work for now 😅